tests/unit/crawlers/_adaptive_playwright/test_adaptive_playwright_crawler.py

```diff
@@ -74,6 +76,13 @@ def test_urls(respx_mock: respx.MockRouter) -> list[str]:
     return urls
 
 
+@pytest.fixture
+async def key_value_store() -> AsyncGenerator[KeyValueStore, None]:
+    kvs = await KeyValueStore.open()
+    yield kvs
+    await kvs.drop()
```

Comment:
Just wondering whether to put this into `conftest` and utilize this fixture across different test modules?